org.gitfourchette.gitfourchette.yml

```diff
@@ -0,0 +1,53 @@
+app-id: org.gitfourchette.gitfourchette
+runtime: org.kde.Platform
+runtime-version: '6.7'  # PyQt.BaseApp doesn't support 6.8 yet (https://github.com/flathub/com.riverbankcomputing.PyQt.BaseApp#branch-comparison)
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: '6.7'
+cleanup-commands:
+ - /app/cleanup-BaseApp.sh
+build-options:
+  env:
+    - BASEAPP_REMOVE_WEBENGINE=1
+
+command: gitfourchette
+
+finish-args:
+  - --env=QTWEBENGINEPROCESS_PATH=/app/bin/QtWebEngineProcess
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  # Talk to git remotes
+  - --share=network
+  # Discover ".git" dir associated with files dropped by the user
+  - --filesystem=host:rw
+  # Open directories in system file manager via DBus
+  - --talk-name=org.freedesktop.FileManager1
+  # Start external merge tool via flatpak-spawn
+  - --talk-name=org.freedesktop.Flatpak
```

Comment:
this require an exception

Reply:
Does `--talk-name=org.freedesktop.FileManager1` need an exception? The linter doesn't report a warning about it and it doesn't appear to be mentioned in the [documentation](https://docs.flathub.org/docs/for-app-authors/linter/#exceptions).

(For the second one: got it, I'll submit a PR for `finish-args-flatpak-spawn-access` soon.)

Reply:
No

Reply:
What merge tools are these? Can some be included in the manifest as a default?

Reply:
Out of the box, GitFourchette can drive Meld, KDiff3, SourceGear DiffMerge, Perforce Helix Merge, Beyond Compare, as well as the merge mode in GVim, VS Code, and major JetBrains IDEs.

It's up to the user to pick their favorite, so I didn't add a hard requirement on another package.